alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/binder/solr/sharding/SolrShardingMetricsFactory.java

```diff
@@ -26,6 +31,12 @@ public SolrShardingMetricsFactory(ShardRegistry shardRegistry, MeterRegistry reg
                 .forJob(jobDetail.getKey()).build();
 
         scheduler.scheduleJob(jobDetail, trigger);
+        this.jobDetail = jobDetail;
```

Comment:
Why are we keeping a reference to the `jobDetail` if we don't need it to unregister our job?